R/bainCommon.R

```diff
@@ -672,150 +706,178 @@
+              "hypothesis[type2]" = gettext("H1: Bigger"), "BF[type2]" = "", "pmp[type2]" = tableResults[["PMP_1"]]
+            )
           }
           if (options$hypothesis == "biggerSmaller") {
-            row <-list(Variable=currentPair, "hypothesis[type1]" = gettext("H1: Bigger"), "BF[type1]"= tableResults[["BF_01"]], "pmp[type1]" = tableResults[["PMP_0"]],
-                       "hypothesis[type2]" = gettext("H2: Smaller"), "BF[type2]" = "", "pmp[type2]" = tableResults[["PMP_1"]])
+            row <- list(
+              Variable = currentPair, "hypothesis[type1]" = gettext("H1: Bigger"), "BF[type1]" = tableResults[["BF_01"]], "pmp[type1]" = tableResults[["PMP_0"]],
+              "hypothesis[type2]" = gettext("H2: Smaller"), "BF[type2]" = "", "pmp[type2]" = tableResults[["PMP_1"]]
+            )
           }
           if (options$hypothesis == "equalBiggerSmaller") {
-            row <-list(Variable=currentPair,
-                       "type[equal]" = gettext("H0: Equal"),
-                       "BF[equal]"= "",
-                       "pmp[equal]" = tableResults[["PMP_0"]],
-                       "type[greater]"= gettext("H1: Bigger"),
-                       "BF[greater]" = tableResults[["BF_01"]],
-                       "pmp[greater]" = tableResults[["PMP_1"]],
-                       "type[less]" = gettext("H2: Smaller"),
-                       "BF[less]" = tableResults[["BF_02"]],
-                       "pmp[less]" = tableResults[["PMP_2"]])
+            row <- list(
+              Variable = currentPair,
+              "type[equal]" = gettext("H0: Equal"),
+              "BF[equal]" = "",
+              "pmp[equal]" = tableResults[["PMP_0"]],
+              "type[greater]" = gettext("H1: Bigger"),
+              "BF[greater]" = tableResults[["BF_01"]],
+              "pmp[greater]" = tableResults[["PMP_1"]],
+              "type[less]" = gettext("H2: Smaller"),
+              "BF[less]" = tableResults[["BF_02"]],
+              "pmp[less]" = tableResults[["PMP_2"]]
+            )
           }
         } else if (options$bayesFactorType == "BF10") {
           if (options$hypothesis == "equalNotEqual") {
-            row <- list(Variable=currentPair, "hypothesis[type1]" = gettext("H0: Equal"),"BF[type1]"="", "pmp[type1]" = tableResults[["PMP_0"]],
-                        "hypothesis[type2]" = gettext("H1: Not equal"), "BF[type2]" = tableResults[["BF_0u"]], "pmp[type2]" = tableResults[["PMP_u"]])
+            row <- list(
+              Variable = currentPair, "hypothesis[type1]" = gettext("H0: Equal"), "BF[type1]" = "", "pmp[type1]" = tableResults[["PMP_0"]],
+              "hypothesis[type2]" = gettext("H1: Not equal"), "BF[type2]" = tableResults[["BF_0u"]], "pmp[type2]" = tableResults[["PMP_u"]]
+            )
           }
           if (options$hypothesis == "equalSmaller") {
```

Comment:
Any particular reason why these conditionals are not `if`... `else if`... `else`?

Reply:
Good point, adjusted!